opencensus/proto/metrics/metrics.proto

```diff
@@ -135,19 +119,25 @@ message LabelValue {
 
 // A timestamped measurement.
 message Point {
-  // The moment when this point was recorded.
-  google.protobuf.Timestamp timestamp = 1; // required
+  // Available only for cumulative metrics. The time when the cumulative value
```

Comment:
> Available only for cumulative metrics

Should this be worded more strongly, like "must be present for cumulative metrics."

What happens when it's absent? "Use the previously seen value" gives us back encoding efficiency of moving `start_time` here.

Reply:
done.